lib/jeopardy_web/controllers/game_controller.ex

```diff
@@ -12,14 +14,44 @@ defmodule JeopardyWeb.GameController do
   end
 
   def create(conn, %{"game" => game_params}) do
+    game_params = %{name: "random"}
     with {:ok, %Game{} = game} <- Games.create_game(game_params) do
+      categories = create_categories_from_api(game)
+      IO.puts("HEY YO #{Kernel.inspect(categories)}")
       conn
       |> put_status(:created)
       |> put_resp_header("location", game_path(conn, :show, game))
       |> render("show.json", game: game)
     end
   end
 
+  def create_categories_from_api(game) do
+    offset = Kernel.inspect(:rand.uniform(100))
+    url = "http://jservice.io/api/categories?count=5&offset=" <> offset
+    req = Poison.decode!(HTTPoison.get!(url).body)
+    IO.puts("RESULT EY #{Kernel.inspect(req)}")
+    categories = Map.new()
+    for cat_req <- req do
+      cat_params = %{"title": cat_req["title"]}
+      with {:ok, %Category{} = category} <- Games.create_category(cat_params) do        
+        cat_url = "http://jservice.io/api/category?id=" <> Kernel.inspect(cat_req["id"])
+        IO.puts("CAT #{Kernel.inspect(category)}")
+        cat = Poison.decode!(HTTPoison.get!(cat_url).body)
```

Comment:
does the api not let us preload the categories in a single request? :(

Reply:
yeah too bad they don't have that
![image](https://user-images.githubusercontent.com/6301013/38586971-d96325c0-3ced-11e8-88fe-37e097a59bf2.png)